app/api/hateoas/HateoasLinks.scala

```diff
@@ -16,34 +16,34 @@
 
 package api.hateoas
 
+import api.hateoas
```

Comment:
This line can be removed if we use 'Link(href = annualSubmissionUri(appConfig, nino, businessId, taxYear), method = GET, rel = SELF)' we do not need to use "hateoas."